cmd/svcat/binding/unbind_cmd.go

```diff
@@ -83,6 +107,41 @@ func (c *unbindCmd) Run() error {
 	var bindings []types.NamespacedName
 	var err error
 
+	if c.abandon {
+		fmt.Fprintln(c.Output, "This action is not reversible and may cause you to be charged for the broker resources that are abandoned.")
+		if !c.skipPrompt {
+			fmt.Fprintln(c.Output, "Are you sure? [y|N]: ")
+			s := bufio.NewScanner(c.stdin)
+			s.Scan()
+
+			err = s.Err()
+			fmt.Fprintln(c.Output, err)
+
+			if strings.ToLower(s.Text()) != "y" {
+				err = fmt.Errorf("aborted abandon operation")
+				return err
+			}
+		}
+
+		if c.instanceName == "" {
```

Comment:
It's actually possible for there to be multiple bindings to remove, see the bit on line 145 where it calls Unbind or DeleteBindings based on whether instanceName is set or not. I think we also need to handle both cases.